client/src/components/App/Content/UsersManagement/UsersManagement.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import { Grid, TableContainer, Paper, Table, TableHead, TableRow, TableCell, TableBody,
+         IconButton, Tooltip } from '@material-ui/core';
+import { Pagination } from '@material-ui/lab';
+import { PersonPin } from '@material-ui/icons';
+
+import IsActiveToggle from 'commons/IsActiveToggle/IsActiveToggle'
+
+import { UsersManagementTableHeaders, UsersManagementTableHeadersNames } from './UsersManagementTableHeaders'
+import useStyles from './UsersManagementStyles'
+import useUsersManagementTable from './useUsersManagement';
+
+const rowsPerPage: number = 7;
+
+const UsersManagementTable: React.FC = () => {
```

Comment:
The component name should be the same as the file name.

Reply:
fixed